ircspeak.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-shuf -n 1 ~/irclogs/freenode/\##zuzakistan.log
+shuf -n 1 ~/irclogs/freenode/$1.log
```

Comment:
Should the path be configurable?

Reply:
yes, it shouldn't even be Bash stuff
we'll probably fix it in like 2023